docs/content/docs/build/releases-and-apitags.md

```diff
@@ -37,20 +37,20 @@ There are no guarantees for API breakages or Fluid document compatibility among
 
 ## API Tags
 
-There are no API stability guarantees for packages in the `@fluid-experimental` and `@fluid-internal` scopes.
-`@fluid-experimental` APIs are for developers to try experimental features and provide feedback. It's possible that these APIs can get completely scrapped or drastically changed in a minor release based on developer feedback.
-`@fluid-internal` APIs are only meant for internal consumption by the framework and should not be used.
-
```

Comment:
Suggest moving this section to the end, since it's relevant for the smallest section of the audience (and fairly self explanatory.)